### PR TITLE
test(samples): added postman sample to start the regression tests

### PR DESCRIPTION
**Pull request by ktfth**
added samples to help automate regression tests
added instructions on own readme of the sample and the main readme added the usage of newman a cli tool from nodejs ecosystem

### CODE REVIEW

File: samples/README.md
Comment (avelino) [inline samples/README.md:1]: This page could be inside the documentation, I don't think it is good to have readme in sub folder, it having to be forgotten to be updated in the future

**Review by avelino — CHANGES_REQUESTED** [review]

File: samples/README.md
Comment (ktfth) [reply samples/README.md:1]: I can put the information on doc, but you see the possibility to maintain the collection and use as part of a regression test to be used on the process of CI/CD to guarantee the actual working state of the project?

Comment (ktfth) [reply samples/README.md:1]: @avelino any considerations about that? Makes sense to start this kind of test to maintain the compatibility across new versions?

**Comment by avelino** [comment]
@ktfth remember to sign the CLA please

File: samples/README.md
Comment (avelino) [inline samples/README.md:1]: This readme is not needed, the same content is in the documentation

**Review by avelino — CHANGES_REQUESTED** [review]

**Review by avelino — APPROVED** [review]
LGTM

**Comment by arxdsilva** [comment]
Thanks @ktfth !

**Pull request merged**